.gitignore

```diff
@@ -0,0 +1,22 @@
+# Compass / Sass files
+.sass-cache
+
+# OS and Editor files
+.DS_Store
+*.DS_Store
+Thumbs.db
+*.sublime-project
+*.sublime-workspace
+.svn
+.idea
+
+# Node modules
+node_modules/
+
+# Adobe Asset Files
+*.psd
+*.ai
+*.indd/vendor/
+
+# Assets generated by build pipeline
+static/assets/*
```

Comment:
These assets are generated by `gulp` so we shouldn't need them in version control. They also exist in the `assets/*` directories in their uncompressed, uncompiled, state.